main.tf

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Terraform and AWS Provider blocks
+# ----------------------------------------------------------------------------------------------------------------------
+terraform {
+  # This module is now only being tested with Terraform 0.13.x.
+  required_version = ">= 0.13.0"
+}
+
+provider "aws" {
+  region  = var.region
+  profile = var.aws_profile
+  version = "= 3.3.0"
+}
+# ----------------------------------------------------------------------------------------------------------------------
+# Locals and Data Sources
+# ----------------------------------------------------------------------------------------------------------------------
+locals {
+  prefix = "${var.env}-${var.project_name}-${replace(data.aws_region.current.name, "-", "")}"
```

Comment:
I think we should not prefix module resources with `var.env`, `var.project_name` and region because it's a reusable module.
Instead of that, we can delete `var.env`, `var.project_name` inputs and replace it with a `var.prefix` that can be specified by the caller module